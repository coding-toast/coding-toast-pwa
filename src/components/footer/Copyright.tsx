import * as React from 'react';
import { Col } from 'react-bootstrap';

const Copyright: React.FC = () => (
  <Col className='text-center'>
    <small className='text-muted'>
      &copy; {new Date().getFullYear()} Coding Toast
      <br />
      Quentin Guenther, Nathan Corbin
    </small>
  </Col>
);

export default Copyright;
