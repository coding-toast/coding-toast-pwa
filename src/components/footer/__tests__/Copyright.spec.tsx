import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Copyright from '../Copyright'

describe('<Copyright />', () => {
  it('renders correctly', () => {
    const component = shallow(<Copyright />)

    expect(toJson(component)).toMatchSnapshot()
  })
})
