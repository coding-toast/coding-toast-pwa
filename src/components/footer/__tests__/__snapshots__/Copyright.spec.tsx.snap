// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Copyright /> renders correctly 1`] = `
<Fragment>
  <p>
    Â© 
    2020
     Coding Toast
    <br />
    Quentin Guenther, Nathan Corbin
  </p>
</Fragment>
`;
